src/reducer/uptime.js

```diff
@@ -12,6 +12,28 @@ const uptime = (state=initialState, action)=> {
   }
 
   switch (action.type) {
+    // Get dashboard stats
+    case ActionTypes.GET_DASHBOARD_STATS_REQUEST_ATTEMPTED: {
+      return {
```

Comment:
Can we just do `return state;` instead ?

Reply:
Sure, will modify the code accordingly.